xml/core_registers.xml

```diff
@@ -67,22 +97,30 @@
             is running.
         </field>
         <field name="stopcount" bits="10" access="WARL" reset="Preset">
-            0: Increment counters as usual.
+            <value v="0" name="increment">
+            Increment counters as usual.
+            </value>
 
-            1: Don't increment any hart-local counters while in Debug Mode or
+            <value v="1" name="freeze">
+            Don't increment any hart-local counters while in Debug Mode or
             on {\tt ebreak} instructions that cause entry into Debug Mode.
             These counters include the {\tt instret} CSR. On single-hart cores
             {\tt cycle} should be stopped, but on multi-hart cores it must keep
             incrementing.
+            </value>
 
             An implementation may hardwire this bit to 0 or 1.
         </field>
         <field name="stoptime" bits="9" access="WARL" reset="Preset">
-            0: Increment \Rtime as usual.
+            <value v="0" name="ignore">
```

Comment:
"ignore" seems strange.  Should it be "increment" like stopcount above?  If "increment" is not ideal then maybe there's a common term that is appropriate for both stopcount and stoptime (like "normal" or something that implies that it behaves as specified in the non-debug specs.

Reply:
increment feels weird. Clocks don't increment. How about run/freeze? Does run/freeze make sense for stopcount?

Reply:
Yes, increment seems weird.  Counters don't even necessarily increment if they're counting something that never happens.  normal/freeze?

Reply:
Done.